src/v0/destinations/snapchat_custom_audience/networkHandler.js

```diff
@@ -63,16 +62,34 @@ const scAudienceProxyRequest = async (request) => {
 
 const scaAudienceRespHandler = (destResponse, stageMsg) => {
   const { status, response } = destResponse;
-  // const respAttributes = response["@attributes"] || null;
-  // const { stat, err_code: errorCode } = respAttributes;
+  const authErrCategory = getAuthErrCategory(status, response);
+
+  if (authErrCategory === REFRESH_TOKEN) {
+    throw new RetryableError(
+      `Failed with ${response} ${stageMsg}`,
+      HTTP_STATUS_CODES.INTERNAL_SERVER_ERROR,
```

Comment:
it is fine for now but don't have any other error code for refresh token?

Reply:
If you're talking in terms of identifier, we are using `REFRESH_TOKEN` as `authErrorCategory` & since the retrial of event has to happen, we are sending `500` as the response status-code